src/igs_network.c

```diff
@@ -4737,6 +4748,11 @@ void igs_set_ipc_dir (const char *path)
                 core_context->network_ipc_folder_path = strdup (path);
             } else if (core_context->network_ipc_folder_path)
                 igs_error ("IPC dir remains set to %s", core_context->network_ipc_folder_path);
```

Comment:
igs_error ("IPC folder remains set to %s", core_context->network_ipc_folder_path);